### PR TITLE
Prepare for nu pkg harvesting

### PR DESCRIPTION
**Pull request by ericstj**
Update buildtools with support for harvesting package content from previous packages without using depproj.

/cc @weshaggard @joperezr

### CODE REVIEW

**Comment by ericstj** [comment]
(Waiting for this buildtools build so expect CI to fail)

**Comment by ericstj** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop CentOS7.1 Release Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop Ubuntu14.04 Release Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop Ubuntu14.04 Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop Ubuntu14.04 Debug Build and Test

File: pkg/Microsoft.Private.PackageBaseline/project.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
+{
+  "frameworks": {
+    "netstandard1.6": {
```
Comment (joperezr) [inline pkg/Microsoft.Private.PackageBaseline/project.json]: I suppose we plan to generate this manually every time we ship a netstandard version?

Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/project.json]: For now it's checked in.  Wes and I were thinking about ways to generate it during a restore, along with generating other project.jsons :smile:  not handling that now.

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**